apps/web/app/constants.ts

```diff
@@ -0,0 +1,24 @@
+import packageJson from '../package.json';
+
+export const MICROBLOCKS_ENABLED = true;
+
+export const IS_BROWSER = typeof document !== 'undefined';
+
+export const DEFAULT_V2_INFO_ENDPOINT = '/v2/info';
+
+export const APP_DETAILS =
+  typeof window !== 'undefined'
```

Comment:
Don't understand this. It's a server/client check, how does window not being defined indicate testing env?

Reply:
copy pasted it, checking now, it's not even user, will remove all unused consts